src/weapon.py

```diff
@@ -13,14 +13,12 @@ class Weapon:
     input_button: VirtualButton
     car: arcade.Sprite
     bullet_list: arcade.SpriteList
-    button_held: bool
```

Comment:
Removed this from `Weapon` since it's no longer something that all (most?) weapons need.

Reply:
Agreed